addon/components/suspense-component/index.js

```diff
@@ -0,0 +1,108 @@
+import { getOwner } from "@ember/application";
+import Component from "@glimmer/component";
+import Ember from "ember";
+import IS_BROWSER from "ember-async-component/utils/is-browser";
```

Comment:
It's best to import relative here I think:

```suggestion
import IS_BROWSER from "../../utils/is-browser";
```